docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+services:
+  db:
+    image: postgres:12.3
+    volumes:
+      - db_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: "postgres"
+      POSTGRES_PASSWORD: "postgres"
+    ports:
+      - "5432:5432"
+
+  redis:
+    image: redis:5.0
+    volumes:
+      - redis_data:/redis_data
+    ports:
+      - "6379:6379"
+
+  authgear:
+    build: .
+    environment:
+      DEV_MODE: "true"
+      LOG_LEVEL: "debug"
+    volumes:
+      - ./authgear.yaml:/authgear.yaml
+      - ./authgear.secrets.yaml:/authgear.secrets.yaml
```

Comment:
The WORKDIR is `/app` so we should mount them as `/app/authgear.yaml` and `/app/authgear.secrets.yaml`.